index.js

```diff
@@ -1,13 +1,44 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
-console.log("Here's your header:", header)
-
-
 /***** Deliverable 2 *****/
 
+const header = document.querySelector('h1').style.color = 'red'
```

Comment:
This works, but try to separate code like this onto a couple of lines, like this, for readability:

```js
const header = document.querySelector('h1')
header.style.color = 'red'
```